frida_tools/lsd.py

```diff
@@ -19,9 +19,20 @@ def _start(self) -> None:
                 self._update_status(f"Failed to enumerate devices: {e}")
                 self._exit(1)
                 return
+            device_os = {}
+            for device in devices:
+                try:
+                    params = device.query_system_parameters()
+                    if "name" in params:
+                        device.name = params["name"]
+                    if "os" in params:
```

Comment:
No need to check for `os`, as it's required. There's a "schema" [here](https://github.com/frida/frida-node/blob/922d43880468465f84e31c63d8d0fb031030d119/lib/system_parameters.ts#L5-L20).